### PR TITLE
Initial pages

### PR DESCRIPTION
**Pull request by Zejnilovic**
I have created initial pages for the project and for some open positions we have. Please check.

- Assets remain as in the Enceladus GH pages
- Removed index.html, replaced with index.md
- new collection careers with our career options

## To run on localhost
```bash
bundle install
bundle exec jekyll serve
```

### CODE REVIEW

File: atum.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Atum
```
Comment (felipemmelo) [inline atum.md]: ABRiS seems to be missing.

Comment (Zejnilovic) [reply atum.md]: It is. Will add.

Comment (Zejnilovic) [reply atum.md]: Added. Nice spot. You could be a QA 👍

File: _careers/big_data_eng.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### Must Have
+- Strong problem solving skills
+- Strong programming skills
+- Background in computer science, engineering, physics, mathematics or equivalent
+- Be eager to learn new approaches and technologies
+
+### Desirable Skills
+- Experience with Scala or other functional languages (Haskell, Clojure, Kotlin, Clean) 
+- Experience with any of the
```
Comment (GeorgiChochov) [inline _careers/big_data_eng.md]:
```suggestion
- Experience with any of the following:
```

File: _careers/big_data_eng.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Big Data Engineer
+location: "Prague, CZ"
+excerpt: "Big Data Engineer will work alongside of experienced and junior engineers on development of mission critical projects and applications using the most modern Big Data technology stack."
```
Comment (GeorgiChochov) [inline _careers/big_data_eng.md]:
```suggestion
excerpt: "As a Big Data Engineer you will work alongside experienced and junior engineers on development of mission critical projects and applications using the most modern Big Data technology stack."
```

File: spline.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Spline
+---
+
+For now, please refer to the GitHub repository of [Spline](https://github.com/AbsaOSS/Spline)
```
[the remark this answers is not in the document]
Comment (Zejnilovic) [reply spline.md]: @wajda What would you like to have there for now?

**Pull request merged**